src/os/bluestore/BlueStore.h

```diff
@@ -2017,7 +2017,6 @@ class BlueStore : public ObjectStore,
   private:
     void _adjust_cache_settings();
     void _resize_shards(bool interval_stats);
-    void _tune_cache_size(bool interval_stats);
     void _balance_cache(
```

Comment:
I believe we can also remove _balance_cache and _balance_cache_pri here

Reply:
OK, will do